server/index.js

```diff
@@ -68,6 +68,9 @@ api.get('/test', findUserWithPhoneNumber, addSubscriptionToTable);
 
 app.use('/api', api);
 
+app.get('/api/test', findUserWithPhoneNumber, addSubscriptionToTable);
```

Comment:
I wondered about that too, but I'm not sure how it, I just saw it and figured that it was testing someone but didn't want to delete something that maybe is still being used even though it's for testing